### PR TITLE
little punctuation and add a phrase

### PR DESCRIPTION
**Pull request by mbaykara**
Add the `second` to improve readability.

### CODE REVIEW

File: docs/markdown/Running-Meson.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ short-description: Building a project with Meson
 
 There are two different ways of invoking Meson. First, you can run it
 directly from the source tree with the command
-`/path/to/source/meson.py`. Meson may also be installed in which case
+`/path/to/source/meson.py`. Second, Meson may also be installed in which case
```
Comment (eli-schwartz) [inline docs/markdown/Running-Meson.md:9]: When changing from "Meson may also be installed"

shouldn't it become

"Second, Meson may be installed"?

Comment (mbaykara) [reply docs/markdown/Running-Meson.md:9]: I would say no, but I am not sure 100%.

Comment (dcbaker) [reply docs/markdown/Running-Meson.md:9]: I don't think "second" is really the right word here. Maybe "Alternatively, Meson may be installed, in which case…"

**Comment by dcbaker** [comment]
Otherwise this looks like nice changes to me.

**Pull request merged**